commands.js

```diff
@@ -701,6 +701,9 @@ var commands = exports.commands = {
 			}
 			return connection.sendTo(target, "|noinit|joinfailed|You do not have permission to join '" + target + "'.");
 		}
+		if (toId(target) in Rooms.aliases) {
+			connection.send(">" + toId(target) + "\n|deinit");
+		}
```

Comment:
The way I'd been trying to get Morf to do it was `|noinit|alias|<actual roomid>` (and have the room replacement be seamless clientside)